web/index.html

```diff
@@ -13,27 +13,27 @@
   }
   </script>
 
-  <body class="avenir cover" onload="enableBtn()">
-  <!-- body class="avenir cover" onload="disableBtn()"-->
-
-    <article class="dt w-100 vh-100">
-
-      <div class="dtc v-mid tc center pa2">
-
-        <div class="bg-green tc bn br3 center measure pa2 pt4" id="thankyou" style="display: none">
-          <div class="pb4">
-            <span class="avenir white fw4 f5">Thank you for signing up to our Slack Community. An invite should be heading your way shortly.</span>
-          </div>
-          <div class="pb4">
-            <a class="pa2 ph5 bn no-underline dark-green fw6 bg-white grow" href="index.html">Done</a>
-          </div>
-        </div>
-
-
+<body class="w-100 sans-serif bg-black" onload="enableBtn()">
+<main>
+  <div data-name="component">
+    <section class="flex-ns vh-100 items-center">
+      <div class="mw6 ph5 bg-black">
+        <img src="./iphone-slack.png">
+      </div>
+      <div class="tc tl-ns ph3">
+        <h1 class="near-white f1">
+          <span>STACKSTORM'S&nbsp;</span>
+          <span><font color="#ff9212">OPEN SOURCE <br />COMMUNITY</font>
+          RUNS ON SLACK</span>
+        </h1>
+        <h2 class="f5 tc f3-l fw1 mb4 near-white lh-title">Welcome to StackStorm! </h2>
+        <p class="f5 tc f3-l fw1 near-white lh-title">
+        Please introduce yourself, and leave an email <br /> to receive an invite to our Slack community.
```

Comment:
I meant "introduce yourself" right here on the form, as a way to say "enter your name". The "name" field is also desirable. I can make it optional as @alexmakarski suggested, but somehow still care for a name.